libraries/SQLiteCpp/.gitignore

```diff
@@ -1,6 +1,7 @@
 Debug
 Release
 build
+example1
 *.a
```

Comment:
Did you do this on purpose?

Reply:
No, must have been created by Clion

Reply:
i think i deleted this line from .gitignore in some commit. Do you have the latest version merged ? Also update to googletest looks suspicious ?